src/Plugin.php

```diff
@@ -54,13 +54,6 @@ private function getBundleStubs(): array
             return [];
         }
 
-        if (
```

Comment:
shouldn't it be included all the time now that we can't have less than 1.3.0?

Reply:
Yeah, that's why I moved the contents of `persistence-1.3+/ServiceEntityRepository.phpstub` into `ServiceEntityRepository.phpstub`.

Reply:
Right, I didn't notice that. Can you remove the file and the directory in your PR then?

Reply:
I already did that. 😅

Reply:
Not used to see deletions in github diff sorry 😅